spec/models/user_spec.rb

```diff
@@ -4,7 +4,6 @@
   describe 'validations' do
     it { should validate_presence_of(:name) }
     it { should validate_presence_of(:email) }
-    it { should validate_presence_of(:bio) }
```

Comment:
We removed this validation because some people might not want a bio (it's usually optional on other sites). Also, because it was giving us trouble when creating an account with Google.